spec/features/user_views_index_page_spec.rb

```diff
@@ -0,0 +1,34 @@
+# require "spec_helper"
+#
+# feature "user views a list of parks" do
+#   scenario "user visits " do
+#
+#     feature "visitor sees a list of restaurants" do
+#       scenario "sees a list of restaurants and link for new restaurant" do
+#         dumpling_king = Restaurant.create(name: 'Dumpling King', address: '33 Harrison Ave', city: 'Boston', state: 'MA', zip: '02111')
+#         rock_bottom = Restaurant.create(name: "Rock Bottom", address: "Beach streer", city: "Boston", state: "MA", zip: "02112")
+#
+#         visit restaurants_path
```

Comment:
Also, make sure you change the code when you do submit this. It's copied from another test and says "restaurant" instead of "parks"...